integration_tests/market.feature

```diff
@@ -1,12 +1,5 @@
 Feature: Market Tests
 
-Scenario: Market stats endpoint is functional
-  Given redis container is started
-  And d3a is started using setup strategy_tests.external_market_stats (-t 1s -s 60m)
-  And the external client is started that connects to house-2 market
-  When the external client is requesting market stats
-  Then the external client does not report errors
-
```

Comment:
Instead of deprecating this test altogether you could test the `dso_market_stats`. 
Or do you think it is tested thoroughly in the aggregator tests already (which is the way forward) ?